Makefile

```diff
@@ -1,5 +1,6 @@
 ifeq ($(OS),Windows_NT)
 	STDERR=con
+	EXEEXT=.exe
```

Comment:
where did that come from?

Reply:
@klaernie Windows' exe's need an extension for Windows to execute them.